quarkus/config-api/src/main/java/org/keycloak/config/TruststoreOptions.java

```diff
@@ -0,0 +1,18 @@
+package org.keycloak.config;
+
+import org.keycloak.common.enums.HostnameVerificationPolicy;
+
+public class TruststoreOptions {
+
+    public static final Option<String> TRUSTSTORE_PATHS = new OptionBuilder<>("truststore-paths", String.class)
+            .category(OptionCategory.TRUSTSTORE)
+            .description("List of pkcs12 (.p12 or .pfx), PEM files, or directories containing those files that will be used as a global truststore.")
```

Comment:
Noticed one last issue. Seems the file extensions render weirdly:
<img width="419" alt="Screenshot 2023-11-28 at 12 56 46" src="https://github.com/keycloak/keycloak/assets/13906855/45cf272e-45ff-4018-a1a2-17258522b9fd">
This is due to how we handle dots in options descriptions:
https://github.com/keycloak/keycloak/blob/bfcc278deb5cbe949078508d22320f6689b64497/docs/maven-plugin/src/main/java/org/keycloak/guides/maven/Options.java#L149-L156
I think easiest is just to workaround it for now:
```suggestion
            .description("List of pkcs12 (p12 or pfx file extensions), PEM files, or directories containing those files that will be used as a global truststore.")
```